compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/KernelDispatch.cpp

```diff
@@ -295,12 +305,15 @@ static LogicalResult setRootConfigForPackPeelPipeline(
   // ------------------------------------------------------
   MLIRContext *context = entryPointFn.getContext();
 
-  // Transpose B matrix from [K N n k] to [K N k n]
+  // For matmul, transpose B matrix from [K N n k] to [K N k n]
```

Comment:
Is this comment correct? 

// For matmul, transpose B matrix from [K N n k] to [K N k n]

Why is B in form [K N n k] to start with, for matmul?

Reply:
This is correct. Pack-transpose is performed after pack operation, and the packing is done by tiling iterator dimensions in their canonical order. Please refer to this https://github.com/llvm/llvm-project/blob/949ef57dd20f8d3f3257376b91af71ab8c380338/mlir/include/mlir/Dialect/Linalg/TransformOps/LinalgTransformOps.td#L761.

Reply:
Wow! Thanks